pkg/controller/csi_handler_test.go

```diff
@@ -1414,11 +1442,12 @@ func TestCSIHandlerReconcileVA(t *testing.T) {
 		{
 			name: "va not attached actual state attached",
 			initialObjects: []runtime.Object{
-				va(false /*attached*/, "" /*finalizer*/, nID /*annotations*/),
+				deleted(va(false /*attached*/, "" /*finalizer*/, nID /*annotations*/)),
+				// va(false /*attached*/, "" /*finalizer*/, nID /*annotations*/),
```

Comment:
good catch, please remove the commented line

Reply:
thanks. done.